draft-ietf-pquip-pqc-hsm-constrained.md

```diff
@@ -238,13 +261,14 @@ There are two distinct approaches to exporting private keys or seeds from a cons
 
 In scenarios where the constrained device has sufficient capability to initiate or terminate a mutually authenticated TLS session, the device can securely transfer encrypted private key material directly to another cryptographic module. 
 
-#### Export to Encrypted File
+####  Export of Encrypted Seeds and Private Keys
 
-In more common constrained device scenarios, for secure exporting of seeds and private keys, a strong symmetric encryption algorithm, such as AES, should be used to encrypt the seed before export. This ensures that the seed remains protected even if the export process is vulnerable to quantum attacks.
+In more common constrained device scenarios, for secure exporting of seeds and private keys, a strong symmetric encryption algorithm, such as AES, must be used to encrypt the seed before export. This ensures that the seed remains protected even if the export process is vulnerable to quantum attacks.
 
-Operationally, the exported data and the AES key should both be protected against unauthorized access or modification.
+Operationally, the exported data and the AES key must both be protected against unauthorized access or modification.
```

Comment:
It is already common in IoT devices to store keying material in off-chip flash and this storage operation automatically performs encryption. An example can be found in the PSA Storage API: https://arm-software.github.io/psa-api/storage/1.0/